server/monitor.go

```diff
@@ -381,7 +381,7 @@ type Varz struct {
 	*Info
 	*Options
 	Port             int               `json:"port"`
-	MaxPayload       int               `json:"max_payload"`
+	MaxPayload       int64             `json:"max_payload"`
```

Comment:
I would be very careful here: this may break user code that embeds/uses public API to process monitoring. I would leave this as `int` (see below)